feature/platform/transceiver/zr_chromatic_dispersion_test/README.MD

```diff
@@ -0,0 +1,51 @@
+# gNMI-1.20: Telemetry: 400ZR Optics Chromatic Dispersion(CD) streaming
+
+## Summary
+
+Validate 400ZR optics module reports Chromatic Dispersion.
+Chromatic Dispersion is frequency dependent change in signal phase velocity due
+to fiber measured in ps/nm 
+
+## Procedure
+
+*   Connect two ZR interfaces using a duplex LC fiber jumper such that TX
```

Comment:
In our lab, we usually have an optical switch in-between DUT ports and traffic generator ports.   Will this be ok for this test?  Any extra steps or changes in pass / fail criteria if we have this?

Reply:
Do you know how much dB loss this switch path introduces in a setup like DUT-PortA = Switch = DUT-PortB
If the loss is 2 - 3dB we should be fine.

Reply:
This b2b ZR link will be un-amplified so as long as we keep this below 3dB of insertion loss we should be fine

Reply:
Updated